index.html

```diff
@@ -3074,97 +3078,30 @@ <h3>WoT Binding Templates</h3>
         over a suitable network protocol. WoT is tackling this
         variety through <a>Protocol Bindings</a>, which must meet a
         number of constraints (see <a href="#sec-protocol-bindings"></a>).
-      </p>
+      </p> -->
       <p> The non-normative <a>WoT Binding Templates</a> specification
         [[?WOT-BINDING-TEMPLATES]] provides a collection of
-        communication metadata blueprints that give guidance on
-        how to interact with different <a>IoT platforms</a>.
-        When describing a particular IoT device or service,
-        the <a>Binding Template</a> for the corresponding <a>IoT Platform</a>
-        can be used to look up the communication metadata that
-        must be provided in the <a>Thing Description</a> to
-        support that platform.
+        blueprints that give guidance on how to interact with different <a>Things</a> 
+        that use different <a>transfer protocols</a>, <a>content types</a> or that are 
+        part different <a>IoT platforms</a> (e.g., OCF, oneM2M, OMA LWM2M, OPC UA) 
+        that require certain combinations of  <a>transfer protocols</a> and 
+        <a>content types</a>. When describing a particular IoT device or service, the 
+        <a>Binding Template</a> for the corresponding <a>IoT Platform</a> can be used
+        to look up the communication metadata that must be provided in the
+         <a>Thing Description</a> to  support that platform.
       </p>
       <figure id="fig-binding-templates">
         <img src="images/bblocks/binding-templates.png" srcset="images/bblocks/binding-templates.svg"
           class="wot-arch-diagram" alt="binding templates" />
         <figcaption>From Binding Templates to Protocol Bindings</figcaption>
       </figure>
       <p> <a href="#fig-binding-templates"></a> shows how <a>Binding
-          Templates</a> are applied. A <a>WoT Binding Template</a>
-        is created only once for each <a>IoT Platform</a> and
-        can then be reused in all <a>TDs</a> for devices of that
-        platform. The <a>Consumer</a> that is processing a <a>TD</a> must implement
-        the required <a>Protocol Binding</a> by including a
-        corresponding protocol stack and by configuring the
-        stack (or its messages) according to the information
-        given in the <a>TD</a>.
-      </p>
-      <p>
-        The communication metadata of
-        <a>Protocol Bindings</a> spans five dimensions:
+          Templates</a> are used. Based on the protocol, media type or platform binding, 
+          a <a>TD</a> is created. The <a>Consumer</a> that is processing a <a>TD</a> must 
+          implement the required <a>Binding Template</a> by including a corresponding 
+          protocol stack, media type encoder/decoder or platforms stack and by configuring
```

Comment:
well some platforms combine both so we need to say something like:

protocol stack or media type encoder/decoder or "protocol stack and media type encoder/decoder together in a certain way".

What I want to say here is that in case of platforms, like Philips Hue, Oracle Digital Twin, WebThings, there is the use of a certain protocol together with media type and data structures. So only with the knowledge of HTTP I can write/consume Hue TDs but they have a pattern.